src/Microsoft.ML.AutoML/AutoMLExperiment/AutoMLExperiment.cs

```diff
@@ -249,19 +250,37 @@ public async Task<TrialResult> RunAsync(CancellationToken ct = default)
             var trialNum = trialResultManager?.GetAllTrialResults().Max(t => t.TrialSettings?.TrialId) + 1 ?? 0;
             var tuner = serviceProvider.GetService<ITuner>();
             Contracts.Assert(tuner != null, "tuner can't be null");
+
             while (!aggregateTrainingStopManager.IsStopTrainingRequested())
             {
-                var setting = new TrialSettings()
+                var trialSettings = new TrialSettings()
                 {
                     TrialId = trialNum++,
                     Parameter = Parameter.CreateNestedParameter(),
+                    StartedAtUtc = DateTime.UtcNow,
```

Comment:
Can this (StartedAtUtc) being tracked in your IMonitor implementation class? For example, maintaining a dictionary of starting UTC for all started trials.

The gist is that we want to keep `TrialSettings` only to save information that is necessary.

Reply:
Sure, I've just removed the `StartedAtUtc` property.